resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -119,6 +119,9 @@
     - name: stats
       label: Statistics
       type: basic:file
+    - name: bigwig_all_gbrowsers
```

Comment:
why not just `bigwig`?

Reply:
To make it more clear that it can be used wit all genome browsers. Probably is redundant. I will change it to `bigwig`.